Storage/SQL/SQLiteBookmarksModel.swift

```diff
@@ -531,6 +531,18 @@ extension SQLiteBookmarks {
             (deleteStructure, args),
         ])
     }
+
+    fileprivate func markBufferBookmarkAsDeleted(_ guid: GUID) -> Success {
+        let insertInPendingDeletions =
+        "INSERT OR IGNORE INTO \(TablePendingBookmarksDeletions) " +
+        "(id, server_modified_at_deletion) " +
+        "SELECT " +
+        "?1 AS id, " +
+        "server_modified AS server_modified_at_deletion " +
+        "FROM \(TableBookmarksBuffer) WHERE guid = ?1"
```

Comment:
I think this is fine, if we turn on proper merging we will never read the PendingBookmarksDeletions table anyway.